client/src/views/Home.vue

```diff
@@ -50,13 +48,18 @@ export default {
   created() {
     this.location_ = getLocationFromRoute(this.$route);
     this.setLocation(this.location_);
+    this.fetchPipelines();
   },
   beforeRouteUpdate(to, from, next) {
     this.location_ = getLocationFromRoute(to);
     next();
   },
   methods: {
-    ...mapMutations(["setLocation"]),
+    ...mapMutations(["setLocation", "setPipelines"]),
+    async fetchPipelines() {
+      const { data } = await this.girderRest.get("viame/pipelines");
+      this.setPipelines(data);
+    },
```

Comment:
If pipelines are global state, I think it would make sense for a `fetchPipelines` action to be in the store.